ftplugin/purescript_pscide.vim

```diff
@@ -227,10 +227,16 @@ function! PSCIDEstart(silent)
 	\ "bower_components/**/*.purs",
 	\ ]
 
+  if executable("spago")
+    let fullCommand = command + [ "$(spago source | cat)" ]
```

Comment:
Also, command substitution does not seem to be expanded: `$(spago source | cat)` ends up verbatim on the `purs ide server` command line.

Reply:
This seems to work instead: `systemlist("spago sources")`

Reply:
Ahh yes - this is much better - thanks!